great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -334,9 +335,8 @@ def _get_batch_definition_list(
                 if batch_parameters_counts[param_name] == 1:
                     batch_request_copy_without_partitioner_kwargs.options.pop(param_name)
                 else:
-                    warnings.warn(
-                        f"The same option name is applied for your batch regex and partitioner config: {param_name}"  # noqa: E501
-                    )
+                    # TODO: figure out what to do here!
```

Comment:
This checking was a result of issues between sorters for batching regex and partitioners. This issue should be resolved in a followup PR where I remove `add_sorters`